kep/206-resource-limit-aware-scoring/README.md

```diff
@@ -0,0 +1,96 @@
+# Resource limit-aware node scoring
+
+## Table of Contents
+
+<!-- toc -->
+- [Node resources allocatable scoring](#node-resources-allocatable-scoring)
+  - [Table of Contents](#table-of-contents)
+  - [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+  - [Use Cases](#use-cases)
+  - [Terms](#terms)
+  - [Proposal](#proposal)
+  - [Design Details](#design-details)
+    - [Extension points](#extension-points)
+      - [Score](#score)
+  - [Known Limitations](#known-limitations)
+  - [Alternatives considered](#alternatives-considered)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Testing Plan](#testing-plan)
+  - [Implementation History](#implementation-history)
+  - [References](#references)
+<!-- /toc -->
+
```

Comment:
The template is expected to have a `Summary` section, isn't it?

Reply:
> The template is expected to have a `Summary` section, isn't it?

Added a summary section.